### PR TITLE
livenessProbe in kube-dashboard is configurable

### PR DESCRIPTION
**Pull request by ljfranklin**
- Adds configurable `livenessProbe.initialDelaySeconds` and
  `livenessProbe.timeoutSeconds`

Signed-off-by: Lyle Franklin <lylejfranklin@gmail.com>

#### What this PR does / why we need it:

The k8s Dashboard takes over a minute to startup on a Raspberry Pi k8s cluster.
My understanding is that the hardcoded 30 livenessProbe delay is restarting the pod before the startup can complete.
This PR makes `livenessProbe.initialDelaySeconds` and `livenessProbe.timeoutSeconds` configurable.

#### Which issue this PR fixes

N/A

#### Special notes for your reviewer:

N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: kubernetes-dashboard
-version: 0.7.3
+version: 0.7.4
```
Comment (davidkarlsen) [inline stable/kubernetes-dashboard/Chart.yaml]:
```suggestion
version: 0.7.5
```

**Comment by ljfranklin** [comment]
@davidkarlsen updated version number as suggested above

**Pull request merged**